src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -104,6 +104,10 @@ public enum OperationKind
         // Newly added
 
         ConditionalAccessExpression,
-        IncrementExpression
+        IncrementExpression,
+
+        Argument,
```

Comment:
If I remember correctly--it's been many years since I knew VB semantics completely--optional byref arguments don't necessarily have a default value in VB. They can be truly omitted and the called method can test to see if they are present.